src/XMakeBuildEngine/Definition/ProjectCollection.cs

```diff
@@ -384,7 +384,7 @@ public static Version Version
                     // Use .CodeBase instead of .Location, because .Location doesn't
                     // work when Microsoft.Build.dll has been shadow-copied, for example
                     // in scenarios where NUnit is loading Microsoft.Build.
-                    s_engineVersion = new Version(FileVersionInfo.GetVersionInfo(FileUtilities.ExecutingAssemblyPath).ProductVersion);
+                    s_engineVersion = new Version(FileVersionInfo.GetVersionInfo(FileUtilities.ExecutingAssemblyPath).FileVersion);
```

Comment:
What does ProductVersion get set to with this new system? Do we anticipate a compat impact here?